reactor3/build.gradle

```diff
@@ -1,3 +1,5 @@
 dependencies {
     api libs.reactor.core
+    implementation libs.context.propagation
+    implementation project(':micrometer-context')
```

Comment:
Would you exclude this `micrometer-context` module from reactor3?
`micrometer-context` has a limitation which isn't supporting the context hook so we cannot include it in reactor3 module by default.

Here's the suggestion:
```
// reactor3/build.gradle
dependencies {
    api libs.reactor.core
}
```

```
// micrometer-context/build.gradle
dependencies {
    implementation libs.context.propagation
    testImplementation project(':reactor3')
}
```
and move all test classes into `micrometer-context` module.

Reply:
@minwoox , thanks for your comments!
I fixed it! 
Please take another look. 🙇‍♂️